### PR TITLE
Update security omniauth version

### PR DESCRIPTION
**Pull request by schweigert**
Hi guys,

This PR update omniauth to `'~> 1.5'`. 1.2 has many vulnerabilities.

- Edit -

It fixes some warnings in test suit.

### CODE REVIEW

File: test/strategy_test.rb [hunk 1]
```diff
@@ -452,7 +452,7 @@ def setup
     end
 
     test 'empty param' do
-      assert_equal nil, strategy.send(:signed_request_from_cookie)
+      assert_nil nil, strategy.send(:signed_request_from_cookie)
```
Comment (gnklausner) [inline test/strategy_test.rb]: Hey, mate. Should this be `assert_nil strategy.send(:signed_request_from_cookie)`?

File: test/helper.rb [hunk 2]
```diff
@@ -41,6 +41,7 @@ def setup
 
     @client_id = '123'
     @client_secret = '53cr3tz'
+    @options = nil
```
Comment (gnklausner) [inline test/helper.rb]: Should this be an empty hash? `{}`?

File: lib/omniauth/facebook/signed_request.rb [hunk 3]
```diff
@@ -8,8 +8,10 @@ class UnknownSignatureAlgorithmError < NotImplementedError; end
 
       attr_reader :value, :secret
 
-      def self.parse(value, secret)
-        new(value, secret).payload
+      class << self
```
Comment (gnklausner) [inline lib/omniauth/facebook/signed_request.rb]: What's the motivation for changing this?

File: .travis.yml [hunk 4]
```diff
@@ -8,11 +8,4 @@ rvm:
   - 2.3.1
   - 2.2.6
   - 2.1
-  - 2.0
```
Comment (gaborszakacs) [inline .travis.yml]: Pls consider updating the [Readme](https://github.com/mkdynamic/omniauth-facebook/blob/master/README.md#supported-rubies) too. `MRI Ruby 1.9.3` support must be dropped as `omniauth-oauth2` v1.5.0 started to use the `%i[]` notation which was introduced only in Ruby 2.0.0, I'm not sure about the others, are those necessary too (in this scope)?

**Pull request closed without merging**